dist/index.html

```diff
@@ -3,17 +3,86 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- font awesome file -->
+    <link rel="stylesheet" href="./assets/css/font-awesome.min.css" />
+
     <link rel="stylesheet" href="./assets/css/main.css" />
     <title>Charity</title>
   </head>
   <body>
     <header>
-      <section class="contact">
-        Welcome text and social media Links
+      <section class="contact bg-c100 text-c700 py-2">
```

Comment:
Good job for using  global style `py-2`
But The height of the contact section is less than the height in the design.
 > Height of contact section in design is : 50px
you just need to increase the padding. to `py-4`

Reply:
Make sure to change `py-2` to `py-4` to match the height in the design